src/app/core/site-header/site-header.component.html

```diff
@@ -46,7 +46,7 @@
               <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
                 <li *ngFor="let lang of languages">
                   <a class="dropdown-item" (click)="useLanguage(lang.code)">
-                    <img src="/assets/images/{{ lang.code }}.svg" alt="" />
+                    <img [src]="lang.code + '.svg' | path: 'imagesPath'" alt="" />
```

Comment:
Template literal zamiast klejenia stringów?

Reply:
Angular Expression Language jest podobny do JS, ale nie jest to JS i m.in. nie wspiera template literałów